### PR TITLE
CC62 show open orders backend improve

### PR DESCRIPTION
**Pull request by sgz886**
### Descriptions
this is a improvement and completion to[ cc26 show list of available orders](https://github.com/courtcanva/cc-franchise/pull/22).

### Link to Jira ticekt here:
[Jira ticket link](https://leonhard.atlassian.net/jira/software/projects/CC/boards/2?selectedIssue=CC-62)

### Acceptance Criteira
1. refine response dto from `OrderGetDto` to `OrderPendingPostDto `(removed redudant attributes)
2.  export pure json data for frontend( vs there are some String previously)), sample:
``` Json
 {
       "id": "11",
        "createdTime": "2022-10-01T20:01:16.395+10:00",
        "postcode": "3000",
        "totalAmount": 999.00,
        "designInformation": {
            "quotation": "quotation",
            "constructionDraw": "https://url",
            "isNeedLevelGround": true,
            "constructionAddress": {
                "country": "Australia",
                "state": "VIC",
                "city": "Melbourne",
                "line1": "1/10 Collins Street",
                "line2": "",
                "postalCode": "3000"
            }
        }
}
```

### Provide screenshots or videos that the function is working.
image
image

### Provide screenshots of no warning, error in console.
image

### Developer Checklist

- [x] No merge conflict with the main branch, make sure your code is up to date with main.
- [x] Check PR title is aligned with your ticket, (i.e [CC-xx: Feature] Create pull request template).
- [x] Check your app will run without crashes.
- [x] Check if there are hard coded values, note them if there are anything.
- [x] No unnecessary spaces between lines of code.
- [x] Check you are using camelCase && PascalCase correctly.
- [x] Check you named your code properly.
- [x] Remove unused comments.

### CODE REVIEW

File: src/test/java/com/courtcanva/ccfranchise/mappers/OrderMapperTest.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package com.courtcanva.ccfranchise.mappers;
+
+import static org.junit.jupiter.api.Assertions.*;
```
Comment (NorrisWu0) [inline src/test/java/com/courtcanva/ccfranchise/mappers/OrderMapperTest.java:3]: Better to [disable wildcard import](https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcard-imports)

File: src/main/java/com/courtcanva/ccfranchise/services/OrderService.java [hunk 2]
```diff
@@ -22,23 +23,22 @@ public class OrderService {
     private final OrderRepository orderRepository;
     private final OrderMapper orderMapper;
 
+    public List<OrderPendingPostDto> getFirstTenOpenOrdersById(Long franchiseeId) {
```
Comment (NorrisWu0) [inline src/main/java/com/courtcanva/ccfranchise/services/OrderService.java:26]: Better to make sure we name out entities consistently, pick either `PendingOrders` or `OpenOrders`

**Review by NorrisWu0 — APPROVED** [review]

File: src/main/java/com/courtcanva/ccfranchise/mappers/OrderMapper.java [hunk 3]
```diff
@@ -1,15 +1,42 @@
...
+import org.mapstruct.Mapping;
+import org.mapstruct.Named;
 import org.mapstruct.ReportingPolicy;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 @Mapper(componentModel = "spring", unmappedTargetPolicy = ReportingPolicy.IGNORE)
 public interface OrderMapper {
+
+    Logger log = LoggerFactory.getLogger(OrderMapper.class);
```
Comment (jiayuu-u) [inline src/main/java/com/courtcanva/ccfranchise/mappers/OrderMapper.java:20]: You can use @Slf4j instead of this

Comment (sgz886) [reply src/main/java/com/courtcanva/ccfranchise/mappers/OrderMapper.java:20]: @slf4j doesn't work in here( Interface).

I used workround according to [stackoverflow](https://stackoverflow.com/a/64404268/14777101)

**Review by jiayuu-u — APPROVED** [review]

**Pull request merged**